docs/superpowers/plans/2026-06-11-web-wallet-e2e-journeys.md

```
@@ -0,0 +1,841 @@
+```ts
+        `E2E_IMPORT_SRP must be a 12/15/18/21/24-word recovery phrase; got ${words.length} word(s).`,
+```
+to:
+```ts
+        `Seed phrase must be a 12/15/18/21/24-word recovery phrase; got ${words.length} word(s).`,
+```
+
+- [ ] **Step 2: Update `.env.e2e.example`**
+
+Replace the block (current lines 21–30):
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)